tests/FSharp.Compiler.Private.Scripting.UnitTests/FSharp.Compiler.Private.Scripting.UnitTests.fsproj

```diff
@@ -17,6 +17,13 @@
   <ItemGroup>
     <ProjectReference Include="$(FSharpSourcesRoot)\fsharp\FSharp.Core\FSharp.Core.fsproj" />
     <ProjectReference Include="$(FSharpSourcesRoot)\fsharp\FSharp.Compiler.Private.Scripting\FSharp.Compiler.Private.Scripting.fsproj" />
+    <ProjectReference Include="$(MSBuildThisFileDirectory)TestProjects\DependentAssemblies\TopAssembly\TopAssembly.csproj"><Private>false</Private></ProjectReference>
+    <ProjectReference Include="$(MSBuildThisFileDirectory)TestProjects\DependentAssemblies\DependentAssembly\DependentAssembly.csproj"><Private>false</Private></ProjectReference>
   </ItemGroup>
 
+  <Target Name="MyCopyFilesToOutputDirectory" AfterTargets="Compile">
+    <Copy SourceFiles="$(OutDir)..\..\..\TopAssembly\$(Configuration)\$(TargetFramework)\TopAssembly.dll" DestinationFolder="$(OutDir)TopAssemblyLocation\" />
```

Comment:
I think it would be more stable to do the assembly copying in the testcase itself to a temporary location, that way it's guaranteed to be sterile.